docs/admininstrator-guide/notification-management.md

```diff
@@ -4,45 +4,43 @@ description: Centrally view all notification subscriptions in your system
 
 # Notification Management
 
-## What are Notifications?
+## What are notifications?
 
-Jira notifications are messages sent to a Mattermost Channel when a particular event occurs in Jira.  They can be subscribed to from a channel via `/jira subscribe` \(managed within Mattermost\) AND/OR a webhook can be manually setup from Jira to send a message to a particular channel in Mattermost \(managed via Jira\).
+Jira notifications are messages sent to a Mattermost channel when a particular event occurs in Jira. They can be subscribed to from a channel via `/jira subscribe` \(managed within Mattermost\). A webhook can can be manually set up from Jira to send a message to a particular channel in Mattermost \(managed via Jira\).
 
-![This is a channel notification of a new bug that was created in Jira](../.gitbook/assets/image%20%281%29.png)
-
-When any webhook event is received from Jira - the plugin reviews all the notification subscriptions and if it matches a rule - it will post a notification to the channel.  If there are no subscription matches, the webhook event is discarded. 
+Notifications and webhooks can be used together or you can opt for one of them.
 
-The notifications and metadata shown in a channel are not protected by Jira permissions.  Anyone in the channel can see what is posted to the channel -  if they click on the issue link, Jira will not let the user see further details if they do not have the appropriate permission.  
+![This is a channel notification of a new bug that was created in Jira](../.gitbook/assets/image%20%281%29.png)
 
-## What is a Notification Subscription?
+When any webhook event is received from Jira the plugin reviews all the notification subscriptions and if it matches a rule it will post a notification to the channel. If there are no subscription matches, the webhook event is discarded.
```

Comment:
```suggestion
When any webhook event is received from Jira, the plugin reviews all the notification subscriptions. If it matches a rule, it will post a notification to the channel. If there are no subscription matches, the webhook event is discarded.
```